### PR TITLE
Revise contributing doc's text

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,18 +6,25 @@ Every little help and credit will always be given.
 There are multiple ways to contribute to Taipy, code, but also reporting bugs, creating feature requests, helping
 other users in our forums, Stack Overflow, etc.
 
-Today the only way to communicate with the Taipy team is by GitHub issues.
+Several channels are open to communicate with the Taipy R&D team:
 
-## Never contributed on an open source project before ?
+- Communicating through the taipy.io forum
```
Comment (FabienLelaquais) [inline CONTRIBUTING.md]: Please make 'taipy.io' a link to where you want people to go.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,18 +6,25 @@ Every little help and credit will always be given.
...
+- Through GitHub issues
```
Comment (FabienLelaquais) [inline CONTRIBUTING.md]: "GitHub issues" should also be a link... to taipy's repo issues

File: docs/contributing/contributing.md [hunk 3]
```diff
@@ -6,20 +6,24 @@ Every little help and credit will always be given.
 There are multiple ways to contribute to Taipy, code, but also reporting bugs, creating feature requests, helping
 other users in our forums, Stack Overflow, etc.
 
-Today the only way to communicate with the Taipy team is by GitHub issues.
+Several channels are open to communicate with the Taipy R&D team:
 
-Before to contribute to Taipy, please read our [Code of conduct](code_of_conduct.md).
+- Communicating through the [taipy.io forum](https://www.taipy.io/community/)
+- Through GitHub issues
```
Comment (FabienLelaquais) [inline docs/contributing/contributing.md]: Same remark.

File: docs/contributing/contributing.md [hunk 4]
```diff
@@ -43,7 +47,7 @@ attached to the `current sprint`. Please, do not work on it, the Taipy team is o
 
 ## Code organisation
 
-Taipy is organised in four main repositories:
+Taipy is organized in four main repositories:
```
Comment (FabienLelaquais) [inline docs/contributing/contributing.md:51]: Please create an issue for this: this section is out-dated.

**Review by FabienLelaquais — APPROVED** [review]
👍

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -6,18 +6,25 @@ Every little help and credit will always be given.
 There are multiple ways to contribute to Taipy, code, but also reporting bugs, creating feature requests, helping
 other users in our forums, Stack Overflow, etc.
 
-Today the only way to communicate with the Taipy team is by GitHub issues.
+Several channels are open to communicate with the Taipy R&D team:
 
-## Never contributed on an open source project before ?
+- Communicating through the [taipy.io forum](https://www.taipy.io/community/)
+- Through [GitHub issues](https://github.com/Avaiga/taipy-doc/issues)
+- etc.
```
Comment (jrobinAV) [inline CONTRIBUTING.md]: I would specify the goal of each channel, and I would add the StackOverflow channel.

**Review by jrobinAV — APPROVED** [review]

File: CONTRIBUTING.md (hunk 5 above)
Comment (ghost) [reply CONTRIBUTING.md]: I don't think I would be able to do these specifications precisely as I'm new to the project. Could you provide me the infos for that part, please ? I'll be happy to insert it where it should be after that.

Comment (jrobinAV) [reply CONTRIBUTING.md]: Sure.

I would recommend using: 
- The forum for any open discussions with the Taipy team.
- A new GitHub issue on the taipy repository to propose an improvement, a new feature, or report a bug.
- A new entry in the GitHub discussion page on the taipy repository to open a discussion on a technical topic related to the implementation of Taipy.
- StackOverflow for precise questions related to the usage of Taipy.

What do you guys think?

Comment (ghost) [reply CONTRIBUTING.md]: I don't know for @florian-vuillemot or @FabienLelaquais but it seems fine to me. Do you have a general Taipy topic URL on Stackoverflow that I can link to that line ?

Comment (ghost) [reply CONTRIBUTING.md]: @jrobinAV I updated the section with your suggestions.

**Pull request merged**